lib/prawn/emoji/index.rb

```diff
@@ -12,7 +12,11 @@ def unicodes_regexp
       private
 
       def build_unicodes_regexp
-        Regexp.compile unicodes.map { |unicode| "\\u{#{unicode}}" }.join('|')
```

Comment:
@xtian At first, you update `emoji/index.yml` to run `bundle exec rake emoji:generate` (The task was added by #10) after `git rebase origin/master`. Therefore, this change is not needed because `emoji/index.yml` should include code of multi-character emoij.